src/main/java/edu/harvard/h2ms/config/SecurityConfig.java

```diff
@@ -19,42 +19,42 @@
 @EnableGlobalMethodSecurity(prePostEnabled = true)
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
-  public SecurityConfig() {
-    super(true);
-  }
-
-  private static final String[] AUTH_WHITELIST = {
-    "/",
-    "/login",
-    "/webjars/**",
-    "/api/passwords/**",
-    "/registration/**",
-
-    // swagger ui
-    "/swagger-resources",
-    "/swagger-resources/**",
-    "/swagger-ui.html",
-    "/v2/api-docs"
-  };
-
-  @Override
-  public void configure(WebSecurity web) throws Exception {
-    web.ignoring().antMatchers(AUTH_WHITELIST).antMatchers(HttpMethod.OPTIONS, "/**");
-  }
-
-  @Override
-  protected void configure(HttpSecurity http) throws Exception {
-    http.authorizeRequests().anyRequest().authenticated();
-  }
-
-  @Bean
-  @Override
-  public AuthenticationManager authenticationManagerBean() throws Exception {
-    return super.authenticationManagerBean();
-  }
-
-  @Bean
-  public PasswordEncoder getEncoder() {
-    return new BCryptPasswordEncoder();
-  }
+    public SecurityConfig() {

```

Comment:
I won't reject the PR for this, but in the future could we try to avoid auto-formatting files? Nothing changed here except whitespace.